pipelines.tf

```diff
@@ -67,7 +67,7 @@ module "cloudtrail_monitoring" {
   pipeline_name = "cloudtrail-monitoring"
   src_repo      = "platsec-cloudtrail-monitoring"
 
-  lambda_function_name = "platsec_cloudtrail_monitoring"
+  lambda_function_name = "orgtrail-monitoring"
```

Comment:
`orgtrail-monitoring`? 

There will be 3 names that refer to same thing 
* platsec-cloudtrail-monitoring
* cloudtrail-monitoring ...and now
* orgtrail-monitoring?

Reply:
I agree. I found that we already have started using the "orgtrail-monitoring" convention within platsec-terraform. In terms of consistency I say this now makes the most sense based upon our current intention with this repo. It's already something we're struggling with.
A good example is the component name vs filename here
https://github.com/hmrc/platsec-terraform/blob/main/components/cloudtrail_monitoring_lambda/orgtrail_sns.tf

In addition, personally, prefixing `platsec` makes little sense in our accounts/repos - it's often fairly redundant. 

My intention is to perform a refactor as such to reduce cognitive load.

Reply:
Thanks for chat @ulaskan going to make changes to pivot and stick with our convention.